homeassistant/components/command_line/binary_sensor.py

```diff
@@ -93,49 +93,32 @@ class CommandBinarySensor(BinarySensorEntity):
 
     def __init__(
         self,
-        hass,
-        data,
-        name,
-        device_class,
-        payload_on,
-        payload_off,
-        value_template,
-        unique_id,
-    ):
+        data: CommandSensorData,
+        name: str,
+        device_class: str | None,
+        payload_on: str,
+        payload_off: str,
+        value_template: Template | None,
+        unique_id: str | None,
+    ) -> None:
         """Initialize the Command line binary sensor."""
-        self._hass = hass
         self.data = data
-        self._name = name
-        self._device_class = device_class
-        self._state = False
+        self._attr_name = name
+        self._attr_device_class = device_class
+        self._attr_is_on = False
```

Comment:
Binary sensors can have a state of STATE_UNKNOWN aka `None` now . Should this start out as `None`?